### ISSUE TITLE
Current issues with OxideFix

### ISSUE THREAD

**Issue by Superstarxalien** [CTR-tools/CTR-ModSDK#19]
- Oxide's character preview in the character select screen is displayed with
the wheels that the other characters' karts have. These are to be removed in
a future version of the mod.
- Oxide's character icon in the character select screen flies into view from
the left side of the screen as opposed to the bottom. It is yet unknown why
this happens.
- Oxide's model in the character select screen displays erroneously in the
European version of the game. This is assumed to be due to how this
functionality was achieved by a manual edit of Oxide's character model, and
thus how model information may be read differently in the European version.
More research on the game's model formats may be required to fix this issue.

**Comment by BlhiteGit** [comment]
OxideFix makes some purple token challenges weird/incompletable, especially  world 4 purple challenge

**Comment by Halfshadow** [comment]
Yes, the TV system as example, and however like you said the audio files would be missing.
